src/styles/block/body.scss

```diff
@@ -0,0 +1,7 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```

Comment:
font styles should always be in `html` selector

Reply:
done